packages/cli/src/commands/alias/ls.ts

```diff
@@ -4,28 +4,27 @@ import table from 'text-table';
 import Client from '../../util/client';
 import getAliases from '../../util/alias/get-aliases';
 import getScope from '../../util/get-scope';
+import { Options, getPaginationOpts } from '../../util/get-pagination-opts';
 import stamp from '../../util/output/stamp';
 import strlen from '../../util/strlen';
 import getCommandFlags from '../../util/get-command-flags';
 import { getCommandName } from '../../util/pkg-name';
-
 import { Alias } from '../../types';
 
-interface Options {
-  '--next'?: number;
-}
-
 export default async function ls(
   client: Client,
   opts: Options,
   args: string[]
 ) {
   const { output } = client;
-  const { '--next': nextTimestamp } = opts;
   const { contextName } = await getScope(client);
 
-  if (typeof nextTimestamp !== undefined && Number.isNaN(nextTimestamp)) {
-    output.error('Please provide a number for flag --next');
+  let validated;
```

Comment:
`validated` is ambiguous and doesn't describe what is validated.
```suggestion
  let paginationOptions;
```